### PR TITLE
Rename GMOS grating

### PR DESCRIPTION
**Pull request by swalker2m**
Renames GMOS "disperser" to "grating".  Sorry for the hassle but it isn't just capriciousness but rather a suggestion/request from Andy.  There's a matching change to `core` but it can be folded in later.

### CODE REVIEW

**Comment by cquiroz** [comment]
Do we need to change ITC to match?

**Comment by swalker2m** [comment]
> Do we need to change ITC to match?

Yeah we probably should.  And now that you mention it, I guess I should update the `ScienceConfigurationModel` as well.

File: modules/core/src/main/scala/lucuma/itc/client/ItcSpectroscopyInput.scala [hunk 1]
```diff
@@ -215,22 +215,22 @@ object ItcSpectroscopyInput {
...
 
   implicit val EncoderScienceConfigurationModel: Encoder[ScienceConfigurationModel] = {
-    case ScienceConfigurationModel.Modes.GmosNorthLongSlit(filter, disperser, fpu, _) =>
+    case ScienceConfigurationModel.Modes.GmosNorthLongSlit(filter, grating, fpu, _) =>
       Json.obj(
         "gmosN" ->
           Json.fromFields(
             List(
-              "disperser"  -> screaming(disperser),
+              "disperser"  -> screaming(grating),
```
Comment (swalker2m) [inline modules/core/src/main/scala/lucuma/itc/client/ItcSpectroscopyInput.scala:223]: Leaving the input until ITC is updated.

**Review by rpiaggio — APPROVED** [review]
LGTM

**Review by cquiroz — APPROVED** [review]
LGTM, does it need core 0.32?

**Comment by swalker2m** [comment]
> LGTM, does it need core 0.32?

No, should be independent.

**Pull request merged**